main.tex

```diff
@@ -93,6 +83,7 @@
     \label{eqn:theipd}
```

Comment:
regarding the comment:

> The payoff matrix
> of PD in the Introduction and the replicator dynamics described as Eq. (25)
> are boorish because each variable is not explained anywhere.

Should we include more discussion regarding the payoff matrix? That `R` stand for the `Reward` payoff etc?

Reply:
Yup! I think I meant to do this but forgot. Thanks.